src/app.ts

```diff
@@ -6,15 +6,31 @@ import { register } from './register'
 import { AnyJson } from "@polkadot/types/types";
 import bodyParser from 'body-parser';
 import locks from "locks";
+import expressRateLimit from 'express-rate-limit';
+
+// per IP rate limit
+const expressIpLimiter = expressRateLimit({
+  windowMs: 48 * 60 * 60 * 1000,
+  max: 3, // Limit each IP to N requests per `window`
```

Comment:
I'm wondering if the limit is not too harsh, since it applies to both failed and successful requests, which have significantly different impact.

Also: What do you think about making the rate limits configurable through env?

Reply:
Yes you are probably right, I think I'll move it into the register call and apply the limiter only after success.
Sure I can make the limits configurable, it might make sense in playground deployments to not have any limits for testing purposes.

Reply:
I opted to keep the limits hard coded, and added one env variable to enable throttling. It will be off by default.